pkg/endpoint/policy.go

```diff
@@ -766,9 +767,12 @@ func (e *Endpoint) Regenerate(owner Owner, reason string) <-chan bool {
 				buildSuccess = false
 				scopedLog.WithError(err).Warn("Regeneration of endpoint program failed")
 				e.LogStatus(BPF, Failure, "Error regenerating endpoint: "+err.Error())
+				owner.SendNotification(monitor.AgentNotifyEndpointRegenerateFail,
+					e.getIDandLabels()+": "+err.Error())
 			} else {
 				buildSuccess = true
 				e.LogStatusOK(BPF, "Successfully regenerated endpoint program due to "+reason)
+				owner.SendNotification(monitor.AgentNotifyEndpointRegenerateSuccess, e.getIDandLabels())
```

Comment:
Same as above:
The return value of this function is being ignored - should we log at debug level if it fails? Potential downside of clogging the logs, though.

Reply:
fixed